.github/workflows/linux-build.yml

```
@@ -0,0 +1,267 @@
+        echo "Configuring build ${{ matrix.build_type }} with gcc"
+        cmake -B build/${{ matrix.build_type }} \
+              -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} \
+              -DCCAP_BUILD_EXAMPLES=ON \
+              -DCCAP_BUILD_TESTS=ON
+    
+    - name: Build
+      run: cmake --build build/${{ matrix.build_type }} --config ${{ matrix.build_type }} --parallel $(nproc)
+    
+    - name: Verify GLFW examples build status
+      working-directory: build/${{ matrix.build_type }}
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2010:warning:2:1: Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames

(shellcheck)